docs/source/user-guide/configuration/index.rst

```diff
@@ -7,7 +7,9 @@ Configuration
 
    use-condarc
    sample-condarc
+   free-channel
```

Comment:
I do not see this file in this PR.  Was it not added to the git repo?

Reply:
It's in a different PR that's supposed to go in today. I'll delete the reference here and fix it when the next PR goes through.